docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -5,7 +5,7 @@
 - A 64-bit machine
 
 - Check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md)
-    - If your system does not have support for SGX1+FLC, simulation mode an be used.
+    - If your system does not support SGX1+FLC,  simulation mode can be used.
```

Comment:
I accidentally add a space before `simulation` 😢

Reply:
@mingweishih BTW, for simple edits, you can just propose a change using the syntax:
````md 
```suggestion
    - If your system does not support SGX1+FLC, simulation mode can be used.
```
````
which will produce an interactive merge option in the PR review like this:

```suggestion
    - If your system does not support SGX1+FLC, simulation mode can be used.
```